tests/unit/test_utils.py

```
@@ -346,50 +347,62 @@ def test_incomplete_sequencing_runs_are_skipped(self):
         Runs to not monitor are determined from the max_age value stored
-        in the config, and mtime of the RunInfo.xml file
+        in the config, and mtime of the RunInfo.xml file.
+
+        Test if they are older than the given age that they are correctly
+        skipped, and if within the given max age they are correctly
+        included.
         """
         sequencer_output_dir = os.path.join(TEST_DATA_DIR, uuid4().hex)
-        old_run = os.path.join(
+        run_dir = os.path.join(
             sequencer_output_dir, "16102023_A01295_001_ABC123"
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)